docker-compose.yml

```diff
@@ -38,7 +40,7 @@ services:
     api:
       build: .
```

Comment:
I don't like that the ip-tables are manipulated by default. Often the user wants to hide the APIs behind a firewall, or at least a URL Proxy (e.g. /pro/api/v2 points to port 8080).

Reply:
Using ports: port:port instead of 127.0.0.1:port:port opens the host's port to the outside world. This should only be done explicitly by the admin and not implicitly by the script.